src/components/Support/ContactSection/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { SupportSection } from "../SupportSection";
+import { ContactCard } from "./ContactCard";
+
+export const ContactSection = () => (
+  <SupportSection
+    id="contact-section"
+    title="Skontaktuj się z nami"
+    card={<ContactCard />}
+  >
+    <p>
+      Masz pytania? Chciałbyś pomóc w inny sposób lub dowiedzieć się więcej o
+      naszej działalności?
+    </p>
+    <div>
+      <p>Skontaktuj się z nami:</p>
+      <p>
+        E-mail: <strong>ak.debica@gmail.com</strong>
```

Comment:
Maybe we could extract `email`, `phone` and `address` as in the `ContactItem` and share it across?